templates/user-data.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
```

Comment:
Could you add some comments here explaining what this does? I think it's not immediately obvious just by looking at this file.